metabrainz/templates/index/funding.json

```diff
@@ -0,0 +1,233 @@
+{
+    "version": "v1.0.0",
+
+    "entity": {
+        "type": "organisation",
+        "role": "owner",
+        "name": "MetaBrainz Foundation Inc",
+        "email": "support@metabrainz.org",
+        "phone": "001-805-459-0815",
+        "description": "The MetaBrainz Foundation is a global community creating and maintaining an open encyclopedia of music and arts metadata. Beginning as a resource for music collectors we currently establish collaborations between volunteers, artists, and industry leaders. Since it’s founding it has grown into a global community of hundreds of thousands of editors collaborating together to build and maintain the MetaBrainz comprehensive datasets.\nThe foundation works within a balanced ecosystem of contributions to the datasets and community and the utilization of the datasets. We encourage all data supporters to participate in the data gathering, editing and peer review processes so that our data can be as comprehensive as possible. Each entity is assigned a permanent MBID (MusicBrainz Identifier) in order to facilitate the creation of a reliable and unambiguous universal lingua franca for music data. MBID’s have opened the possibility of boundless search queries, exceeding the seven traditional identifiers for music data and truly putting knowledge at one's fingertips. Our community aggregated data sets are verified and edited by a global voting system to ensure the data is consistent and correct. We collect information about as many different styles and types of arts and music as possible and do not discriminate when compiling data.\nThe foundation was founded in 2004 by Robert Kaye, in San Luis Obispo, California and is a 501(c)(3) tax-exempt non-profit. Commercial supporters are asked to contribute financially in order to help fund our family of projects, and give back to our community to keep our ecosystem healthy. Non-commercial use of our data remains free, and we are committed to keeping it free to utilize. We are also supported by donations and a number of sponsors who provide funds in order for the foundation to continue to accomplish various project goals. The MetaBrainz Foundation believes in transparent finances, which are published quarterly.",
```

Comment:
This is about MeB but there's a lot of stuff specifically about MusicBrainz in there, should it be more generic / mention BB and whatnot?

Fixed a couple typos and the like, not sure if the suggestion mechanism will show the changes because it seems it's being dumb.

```suggestion
        "description": "The MetaBrainz Foundation is a global community creating and maintaining an open encyclopedia of music and arts metadata. Beginning as a resource for music collectors we currently establish collaborations between volunteers, artists, and industry leaders. Since its founding it has grown into a global community of hundreds of thousands of editors collaborating together to build and maintain the comprehensive MetaBrainz datasets.\nThe foundation works within a balanced ecosystem of contributions to the datasets and community and the utilization of the datasets. We encourage all data supporters to participate in the data gathering, editing and peer review processes so that our data can be as comprehensive as possible. Each entity is assigned a permanent MBID (MusicBrainz Identifier) in order to facilitate the creation of a reliable and unambiguous universal lingua franca for music data. MBIDs have opened the possibility of boundless search queries, exceeding the seven traditional identifiers for music data and truly putting knowledge at one's fingertips. Our community aggregated datasets are verified and edited by a global voting system to ensure the data is consistent and correct. We collect information about as many different styles and types of arts and music as possible and do not discriminate when compiling data.\nThe foundation was founded in 2004 by Robert Kaye, in San Luis Obispo, California and is a 501(c)(3) tax-exempt non-profit. Commercial supporters are asked to contribute financially in order to help fund our family of projects, and give back to our community to keep our ecosystem healthy. Non-commercial use of our data remains free, and we are committed to keeping it free to utilize. We are also supported by donations and a number of sponsors who provide funds in order for the foundation to continue to accomplish various project goals. The MetaBrainz Foundation believes in transparent finances, which are published quarterly.",
```